source/cmain.pas

```diff
@@ -762,8 +762,13 @@ function TMainForm.GetSeries(ANode: TTreeNode; ACaseType: TCaseType;
 procedure TMainForm.GetSocketHandler(Sender: TObject; const UseSSL: Boolean; out
   AHandler: TSocketHandler);
 begin
+  {$IF FPC_FullVersion >= 30200}        // needed for SSL to work in general
+  AHandler := TOpenSSLSocketHandler.Create;
+  TOpenSSLSocketHandler(AHandler).SSLType := stTLSv1_2;

```

Comment:
In that case it'd be better to not enforce a specific `SSLType` as it'll be automatically negotiated by OpenSSL (using `TLS_method`) which will be TLS 1.3 in that case.
```suggestion
  TOpenSSLSocketHandler(AHandler).SSLType := stAny;
```